lessons/11_Lesson11_vortexSourcePanelMethod.ipynb

```diff
@@ -537,14 +545,14 @@
     "    \n",
     "    Returns\n",
     "    -------\n",
-    "    Contribution from panel at point (x, y).\n",
+    "    Contribution from the panel at a given point (x, y).\n",
     "    \"\"\"\n",
-    "    def func(s):\n",
+    "    def integrand(s):\n",
     "        return ( ((x - (panel.xa - numpy.sin(panel.beta)*s))*dxdk\n",
```

Comment:
Hmm, looks like PEP8 agrees with you! I thought I was following the _new_ convention but it's actually the old one... https://www.python.org/dev/peps/pep-0008/#should-a-line-break-before-or-after-a-binary-operator

Either is acceptable, but they think that new code should break before the operator (but the most important thing is to be consistent across a project)